KubeArmor/core/kubeArmor.go

```diff
@@ -338,50 +338,53 @@ func KubeArmor() {
 		dm.Node.KernelVersion = kl.GetCommandOutputWithoutErr("uname", []string{"-r"})
 		dm.Node.KernelVersion = strings.TrimSuffix(dm.Node.KernelVersion, "\n")
 
-		dm.Node.PolicyEnabled = tp.KubeArmorPolicyEnabled
+		kg.Print("Updated the node information")
 
-		cfg.GlobalCfg.Policy = false
-		cfg.GlobalCfg.HostPolicy = true
+	} else if cfg.GlobalCfg.K8sEnv {
+		if K8s.InitK8sClient() {
```

Comment:
We can reduce nesting if the other condition is handled ... 
```suggestion
		if !K8s.InitK8sClient() {
		       print error
		       return
		}
		// ... normal processing
		
```